.github/workflows/run-build.yml

```diff
@@ -0,0 +1,34 @@
+name: Build/release
+
+on:
+  push:
+    tags:
+      - v1.*
+
+jobs:
+  release:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest]
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v1
+
+      - name: Install Node.js, NPM and Yarn
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10
+
```

Comment:
We talked about this today, but I think you're right: Let's also shimmy in a test run before a publish, just to prevent us from shipping a broken app.